test/hocr-pdf/test-hocr-pdf.tsht

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env tsht
+
+work="445442158_0126"
+
+which pdfgrep wget >/dev/null || fail "Test requires 'pdfgrep' and 'wget' installed"
+exec_ok wget --quiet "http://digi.bib.uni-mannheim.de/fileadmin/digi/445442158/hocr/${work}.hocr"
+exec_ok wget --quiet "http://digi.bib.uni-mannheim.de/fileadmin/digi/445442158/max/${work}.jpg"
+hocr-pdf . > "${work}.pdf"
+file_not_empty "${work}.pdf"
+exec_ok pdfgrep 'tribunali' "${work}.pdf"
```

Comment:
Yes, the test could be cleverer. But to install pdfgrep in travis, we'd need sudo and that would mean no container-based infrastructure for Travis CI which slows down the testing.